Detections/AlsidForAD/PrivilegedAccountIssues.yaml

```diff
@@ -0,0 +1,28 @@
+id: a5fe9489-cf8b-47ae-a87e-8f3a13e4203e
+name: Alsid privileged accounts issues
+description: |
+    'Searches for triggered Indicators of Exposures related to privileged accounts issues'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AlsidForAD
+    dataTypes:
+      - AlsidForADLog_CL
+queryFrequency: 2h
+queryPeriod: 2h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+  let SeverityTable=datatable(Severity:string,Level:int) [
+  "low", 1,
+  "medium", 2,
+  "high", 3,
+  "critical", 4
+  ];
+  afad_parser
+  | where MessageType == 0 and (Codename == "C-PRIV-ACCOUNTS-SPN" or Codename == "C-NATIVE-ADM-GROUP-MEMBERS" or Codename == "C-KRBTGT-PASSWORD" or Codename == "C-PROTECTED-USERS-GROUP-UNUSED" or Codename == "C-ADMINCOUNT-ACCOUNT-PROPS" or Codename == "C-ADM-ACC-USAGE" or Codename == "C-LAPS-UNSECURE-CONFIG" or Codename == "C-DISABLED-ACCOUNTS-PRIV-GROUPS")
```

Comment:
Change to List, same as I had shown in "Alsid Active Directory attacks pathways" detection.